composer.json

```diff
@@ -31,7 +34,6 @@
     },
     "suggest": {
         "aws/aws-sdk-php": "Allow using IAM authentication with Amazon ElasticSearch Service",
-        "egeloen/http-adapter": "Allow using httpadapter as transport",
```

Comment:
Was this removed on purpose?

Reply:
yeah, I did it in a separated commit https://github.com/ruflin/Elastica/pull/2066/commits/278ce0dcd628a411caade584c39cabffc8ebb55b (the project is abandoned), but it could be done in another PR.

Reply:
Removal SGTM, can we move it to a separate PR? Sounds like it is not related to the guzzle discussion.